src/mpi/init/initthread.c

```diff
@@ -359,10 +359,30 @@ int MPIR_Init_thread(int *argc, char ***argv, int required, int *provided)
     int exit_init_cs_on_failure = 0;
     MPIR_Info *info_ptr;
 
-    MPIR_T_env_init();
```

Comment:
Here I am removing the line that I added. And I believe I am adding it back in another later commits. Yeah, this is artifact of reordering commits. I'll reconstruct the commits with a fresh PR.